src/server/requestHandlers.js

```diff
@@ -338,6 +348,31 @@ sjrk.storyTelling.server.deleteSingleFileRecoverable = function (fileToDelete, d
     }
 };
 
+fluid.defaults("sjrk.storyTelling.server.clientConfigHandler", {
+    gradeNames: "kettle.request.http",
+    invokers: {
+        handleRequest: {
+            funcName: "sjrk.storyTelling.server.getClientConfig",
+            args: ["{arguments}.0", "{server}.options.globalConfig", "{server}.options.secureConfig"]
+        }
+    }
+});
+
+// Returns a collection of values which are "safe" to share
+// with the client side of the application
+sjrk.storyTelling.server.getClientConfig = function (request, globalConfig, secureConfig) {
+    var clientConfig = {
+        theme: globalConfig.theme || secureConfig.baseThemeName,
+        baseTheme: secureConfig.baseThemeName,
+        authoringEnabled: globalConfig.authoringEnabled
+    };
+
+    request.events.onSuccess.fire({
+        message: "Client configuration request successfully received",
```

Comment:
Indicating `isError: false` for rejected requests should be enough for the client to distinguish success and rejected response. Returning `clientConfig` object should be good enough: `request.events.onSuccess.fire(clientConfig)`.